pep-0001.txt

```diff
@@ -510,7 +509,8 @@ Each PEP should have the following parts/sections:
     ready for consideration are complete and reduces people duplicating
     prior discussion.
 
-12. References -- A collection of URLs used as references through the PEP.
+12. Footnotes -- A collection of footnotes referenced in the PEP, and
+    a central place to list non-inline hyperlink targets.
```

Comment:
> Wordsmithing -- should we avoid "referenced" to avoid confusion with inline references? Perhaps "used"?

Good point; though "used" isn't really accurate either, since the footnotes themselves aren't actually "used" in the PEP, but rather linked. "cited" is a good substitute, though, which I've replaced it with here as well as the template.

> "a central place to list non-inline hyperlink targets" -- do we want to still reccoment [sic] this?

I wouldn't go so far as to say the current language "recommends" (assuming that's the word you mean) this approach, as opposed to simply offers it as an option; I removed "central" both places to put a little less empahsis on this.

I do recommend the "links after each paragraph" approach (that's much more proximate and predictable than after each section) for documentation in the Spyder Documentation Style Guide, but I don't think it makes sense to not at least offer a centralized list as an option, as most current PEPs use it, and it is very useful for PEPs that have large numbers of links or have many links with multiple uses (both of which made it invaluable for PEP 639), to list them in one common, known location.

So I'd favor not being overly prescriptive and breaking with current practice here.